src/components/Table/BodyRow.react.js

```diff
@@ -1,10 +1,23 @@
 import {BodyCell} from "./BodyCell.react";
+import {BodySelectable} from "./BodySelectable.react";
 
 export class BodyRow extends React.Component {
   constructor(props) {
     super(props);
   }
 
+  displaySelectableColumn() {
+    if (this.context.tableStore.getSelectableColumn() !== undefined) {
+      var selectValue = this.props.cells[this.context.tableStore.getSelectableColumn()].toString();
+      return(
+        <BodySelectable
+          value={selectValue}
+          key={selectValue}
```

Comment:
the key is basically used to force the identity of the component to be unique, so that each one can be updated when the component is refreshed